benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -77,6 +77,7 @@ def find_digest(self, timestamp):
 def copy_oss_fuzz_files(project, commit_date, benchmark_dir):
     """Checks out the right files from OSS-Fuzz to build the benchmark based on
     |project| and |commit_date|. Then copies them to |benchmark_dir|."""
+    print('j')
```

Comment:
Is this useful?

Reply:
No

Reply:
OK, I will remove it in the other PR, then.